contracts/RemixReward.sol

```diff
@@ -33,15 +35,18 @@ contract Remix is ERC721, ERC721Enumerable, ERC721Burnable, Ownable {
         types["Contributor"] = true;
     }
 
-    function addType (string calldata tokenType) public onlyOwner {
+    function addType (string calldata tokenType) public {
+        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "only admin");
```

Comment:
not sure if `onlyRole` modifier can be used instead of `require` statement

Reply:
this is what  I've seen in the doc but I can change that ofc.